packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -393,6 +394,13 @@ const FUZZY_EVENTS = new Map([
       event: 'domain_validation_result',
     },
   ],
+  [
+    /^flow\.newsletter\.subscribe$/,
+    {
+      group: GROUPS.newsletters,
+      event: 'subscribe',
+    },
```

Comment:
This doesn't need to be a fuzzy event.  It can be in `EVENTS`.

Reply:
Weird, I thought I updated that. Pushed a fix 👍